internal/pkg/describe/status_test.go

```diff
@@ -101,9 +101,9 @@ func TestWebAppStatus_Describe(t *testing.T) {
 			},
 			mockcwSvc: func(m *mocks.MockalarmStatusGetter) {
 				m.EXPECT().GetAlarmsWithTags(map[string]string{
-					"ecs-project":     "mockProject",
-					"ecs-environment": "mockEnv",
-					"ecs-application": "mockApp",
+					"copilot-application": "mockProject",
+					"copilot-environment": "mockEnv",
+					"copilot-service":     "mockApp",
```

Comment:
Can we change all the mockApp stuff to mockService, and the mockProject text to mockApp?

Reply:
:+1: